bin/o80_log_file_stats

```diff
@@ -31,7 +31,7 @@ def _print_stats(file_path):
         # if 2 observation's iteration are separate by more than 1,
         # it means the logger missed one observation during recording
         if iteration-current_iteration>1:
-            missed_iterations += 1
+            missed_iterations += (iteration-current_iteration)
```

Comment:
I think it should be `interation - current_iteration - 1` as only the iterations up to `iteration` have been missed but not `iteration` itself.